src/server/pfs/server/api_server.go

```diff
@@ -609,7 +608,11 @@ func getFileURL(ctx context.Context, URL string, src Source) (int64, error) {
 		}); err != nil {
 			return err
 		}
-		bytesWritten += int64(fi.SizeBytes)
+		// TODO(2.0 required) - SizeBytes requires constructing the src with
+		// `WithDetails` which we don't do here, so we can't get the size from here
+		// that easily.  One option is to always calculate the size of files in the
+		// source.
+		// bytesWritten += int64(fi.Details.SizeBytes)
```

Comment:
Discussed this case with @brycemcanally, will be handled in a followup PR - the nice thing about this change is it made the places we were using these uninitialized zero-values more apparent, as they'll now segfault.